File: customerview/src/main/java/com/demo/customerview/MainActivity.java
package com.demo.customerview;

import android.graphics.Color;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ImageView;

import com.demo.customerview.customer.CustomDrawable;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ImageView img=findViewById(R.id.img);

        CustomDrawable customDrawable=new CustomDrawable(Color.RED);
        img.setBackground(customDrawable);

    }
}

File: textclock/src/main/java/com/demo/textclock/MainActivity.kt
package com.demo.textclock

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*
import java.util.*
import kotlin.concurrent.timer

class MainActivity : AppCompatActivity() {

    private var mTimer:Timer?=null


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        mTimer= timer(period = 1000){
            runOnUiThread {
                textClock.doInvalicate()
            }
        }
    }

    override fun onDestroy() {
        super.onDestroy()
        mTimer?.cancel()
    }
}
